Package.swift

```diff
@@ -410,6 +403,17 @@ extension Target {
     ]
   )
 
+  static let interoperabilityTestsExecutable: Target = .executableTarget(
+    name: "InteroperabilityTestsExecutable",
```

Comment:
Can we call this `interop-tests`/`interoperability-tests`? Executables are more commonly snake/kebab case yet we never seem to follow that pattern so I'd like us to move in that direction